app/views/assignments/_assignment_form_options.html.erb

```diff
@@ -1,11 +1,19 @@
-<dl class="form">
+<% title_errors = @assignment.errors.full_messages_for(:title).join(", ") %>
```

Comment:
Never used a view model in rails before. Is this kind of what you were thinking?